_site/404.html

```diff
@@ -11,18 +11,18 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <meta name="description" content="Cloud-native business automation for building intelligent applications, backed by battle-tested capabilities.">
-  <link rel="shortcut icon" type="image/png" href="//favicon.ico" >
-  <link href="/assets/css/main.css?1563912273954294000" rel="stylesheet">
+  <link rel="shortcut icon" type="image/png" href="/favicon.ico" >
+  <link href="/assets/css/main.css?1585067818536136137" rel="stylesheet">
 
   <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/all.css" integrity="sha384-lKuwvrZot6UHsBSfcMvOkWwlCMgc0TaWr+30HWe3a4ltaBwTZhyTEggF5tJv8tbt" crossorigin="anonymous">
   <!-- Global site tag (gtag.js) - Google Analytics -->
-  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-125236544-1"></script>
+  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-157945096-1"></script>
```

Comment:
is it needed to get statistics om google analytics?

Reply:
IDK, it automatically changed after I built the website. I agree there are much needed changes in the code itself but this PR is mostly about adding the new content to get started and guides pages. I didn't want to make too many changes at once, to the styling, css, fonts, layouts, etc.. I can remove the metrics completely if you want me to.

Reply:
I was just curios, @sterobin ^